.github/workflows/test-application.yaml

```diff
@@ -216,6 +216,9 @@ jobs:
 
             - name: Install composer dependencies
               uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: 'highest'
+                  composer-options: '--with-all-dependencies'
```

Comment:
actually there is just some newer dev version accidently in the github action cache and composer does not allow it to downgrade it to the older version

Reply:
? 🤔